core/workspace_svg.js

```diff
@@ -978,17 +978,54 @@ Blockly.WorkspaceSvg.prototype.paste = function(xmlBlock) {
 };
 
 /**
+ * Refresh the toolbox unless there's a drag in progress.
+ */
+Blockly.WorkspaceSvg.prototype.refreshToolboxSelection = function() {
+  if (this.toolbox_ && this.toolbox_.flyout_ && !Blockly.Flyout.startFlyout_) {
```

Comment:
Done.
And confirmed, issue submitted on Blockly's issue tracker.
https://github.com/google/blockly/issues/1175